docs: minor clarity and spelling fixes (#185)
a-h/templ: docs/docs/06-static-rendering/02-blog-example.md

* Dancing around the Maypole, a traditional folk activity
* Attending local village fetes and fairs
* Watching or participating in Morris dancing performances

> * Enjoying having a day off from the Early May Bank Holiday

Reviewer: ```suggestion
* Enjoying the public holiday known as Early May Bank Holiday
```

   Watching or participating in Morris dancing performances
  </li>
  <li>

>    Enjoying having a day off from the Early May Bank Holiday

Reviewer: ```suggestion
   Enjoying the public holiday known as Early May Bank Holiday
```
